modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaMicronautServerCodegen.java

```diff
@@ -20,6 +20,7 @@ public class JavaMicronautServerCodegen extends JavaMicronautAbstractCodegen {
     public static final String OPT_CONTROLLER_PACKAGE = "controllerPackage";
     public static final String OPT_GENERATE_CONTROLLER_FROM_EXAMPLES = "generateControllerFromExamples";
     public static final String OPT_GENERATE_CONTROLLER_AS_ABSTRACT = "generateControllerAsAbstract";
+    public static final String OPT_GENERATE_CONTROLLER_NOT_IMPLEMENTED = "generateControllerNotImplemented";
```

Comment:
Can we come up with a more descriptive name for the option. 
Maybe `operationsReturnNotImplementedByDefault` or `generateOperationsToReturnNotImplemented`?

Reply:
Done!